dbstore_test.go

```diff
@@ -505,3 +505,39 @@ func (s *DbZuite) TestSignoffPattern() {
 	require.Equal(s.T(), "3", ws.MustGet("version").String())
 	require.Equal(s.T(), "false", ws.MustGet("is_signedoff").String())
 }
+
+func (s *DbZuite) TestDeprecatedField() {
+	defs := MustNewDefinitions(strings.NewReader(`worksheet some_worksheet {
+		1:field_one text
+		2:field_two text
+		3:field_three text
+	}`))
+
+	store := NewStore(defs)
+
+	ws := defs.MustNewWorksheet("some_worksheet")
```

Comment:
It'd be nicer if this was in the func of run transaction, and only wsId was shared with the next txn to load.